config/app.json

```diff
@@ -6,6 +6,7 @@
     "inline",
     "column-2"
   ],
+  "userStatus":"Speaking",
```

Comment:
This is not clear, it just seems randomly added, I think it needs to go into an `event` object

Reply:
This is how I have represented, is it fine? 
```
"layouts": [
    "classic",
    "inline",
    "column-2"
  ],
  "events":{
    "userStatus":["Attending","Speaking"]
  },
  "isr": {
    "homepage": {
      "revalidateSeconds": 900,
      "description": "15 * 60 = 15 minutes"
    },
```

Reply:
Yes this is better 👍 thank you